app/assets/javascripts/discourse/app/form-kit/components/fk/control-wrapper.gjs

```diff
@@ -0,0 +1,147 @@
+import Component from "@glimmer/component";
+import { concat } from "@ember/helper";
+import { action } from "@ember/object";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import { TrackedObject } from "@ember-compat/tracked-built-ins";
+import FKLabel from "discourse/form-kit/components/fk/label";
+import FKMeta from "discourse/form-kit/components/fk/meta";
+import FormText from "discourse/form-kit/components/fk/text";
+import concatClass from "discourse/helpers/concat-class";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class FKControlWrapper extends Component {
+  get controlType() {
+    switch (this.args.component.name) {
+      case "FKControlRadioGroup":
+        return "radio-group";
+      case "FKControlToggle":
+        return "toggle";
+      case "FKControlInput":
+        return "input-" + (this.args.type || "text");
```

Comment:
For now mostly input type=text is supported, I will add more as I convert more forms